store/sgxLevelDB/sgxLeveldb.go

```diff
@@ -0,0 +1,251 @@
+// Package sgxLevelDB implements a light client level db for panacea-doracle.
+// It does include Set & Get functions that are sealed & unsealed in the sgx environment.
+
+package sgxLevelDB
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/medibloc/panacea-doracle/sgx"
+	log "github.com/sirupsen/logrus"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/errors"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/syndtr/goleveldb/leveldb/util"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+var (
+	// errBatchClosed is returned when a closed or written batch is used.
+	errBatchClosed = errors.New("batch has been written or closed")
+
+	// errKeyEmpty is returned when attempting to use an empty or nil key.
+	errKeyEmpty = errors.New("key cannot be empty")
+
+	// errValueNil is returned when attempting to set a nil value.
+	errValueNil = errors.New("value cannot be nil")
+)
+
+type dbCreator func(name string, dir string) (tmdb.DB, error)
+
+var backends = map[tmdb.BackendType]dbCreator{}
+
+func registerDBCreator(backend tmdb.BackendType, creator dbCreator, force bool) {
+	_, ok := backends[backend]
+	if !force && ok {
+		return
+	}
+	backends[backend] = creator
+}
```

Comment:
Are these operations necessary to initialize leveldb?

Reply:
Thanks, I think your suggestions are reasonable. But, I copy the code from [tendermint/tm-db](https://github.com/tendermint/tm-db/blob/v0.6.7/goleveldb.go), so It is difficult to decide whether it is better to modify it or follow it as it is.

What about your opinion? @gyuguen

Reply:
Aha. Are all of them from tm-db except for the sgx use part?

Reply:
Yes, I just fix some sgx use part.

Reply:
Does the read/write function with LevelDB work well in the SGX environment?
Then I see no problem.

Reply:
Yes, it works well. I perform test that reads the value stored in sgxlevelDB directly, unseals it, and compares it with the original value.